Framework/DataHandling/src/LoadSassena.cpp

```diff
@@ -129,11 +129,9 @@ LoadSassena::loadQvectors(const hid_t &h5file, API::WorkspaceGroup_sptr gws,
   }
 
   qvmod.reserve(nq);
-  auto curr = buf.cbegin();
-  for (int iq = 0; iq < nq; iq++) {
+  for (auto curr = buf.cbegin(); curr != buf.end(); curr += 3) {
```

Comment:
I wasn't sure about the comparability of `const_iterator` and `iterator`, so I searched a bit and it seems to be possible (https://stackoverflow.com/q/35390835 and answers), but maybe for consistency it could be nice to use `buf.cend()`.